### PR TITLE
feat/53 FAQ questions form

### PR DESCRIPTION
**Pull request by abdulmalikyusuf**
Developed an additional questions form where users can submit specific inquiries or questions that are not addressed in the FAQs section. The form includes fields for user's name, email address, phone number, and inquiry. Also implemented form validation.

[Figma link](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=432-7537&m=dev)

localhost_5173_

### CODE REVIEW

**Comment by chumex412** [comment]
Resolve the conflicts

File: app/components/faq/form.tsx
Comment (SirhmVFX) [inline app/components/faq/form.tsx:1]: Use global colors, don't hardcode the colors

**Review by SirhmVFX — CHANGES_REQUESTED** [review]

File: app/components/faq/form.tsx
Comment (abdulmalikyusuf) [reply app/components/faq/form.tsx:1]: Okay, will change that

Comment (abdulmalikyusuf) [reply app/components/faq/form.tsx:1]: Please can you add this variable to the globals css file: --primary: 25 95% 53% //hsl value. I need it as the primary color'

**Comment by oreoluwa212** [comment]
I dont see your tests screenshots

**Review by oreoluwa212 — CHANGES_REQUESTED** [review]
Add a video too of your components working after you pulled from dev

**Comment by abdulmalikyusuf** [comment]
> I dont see your tests screenshots

From the task posted on slack, only the next.js repo was to be tested and the packages for testing (vitest and others) are not included in the repository

**Pull request closed without merging**